server/tests/test_app.py

```diff
@@ -12,6 +12,11 @@ def test_index(client: FlaskClient):
     assert b"Arlo, by VotingWorks" in rv.data
 
 
+def test_logo(client: FlaskClient):
```

Comment:
needed to increase coverage and my files were already fully covered, so thought this was a useful added test to add one more line of coverage.